lib/validation.js

```diff
@@ -7,15 +7,19 @@ const {
   kSchemaBody: bodySchema,
   kSchemaResponse: responseSchema
 } = require('./symbols')
+const scChecker = /^[0-9]{3}$|^[0-9]xx$/
```

Comment:
```suggestion
const scChecker = /^[1-5]{3}$|^[1-5]xx$/
```
There is no status code `0xx` and `[6-9]xx`

Reply:
I was being a tad lazy 😅